### DISCUSSION TITLE
Dyna input creates incorrectly formatted engine file

### DISCUSSION THREAD

**Discussion by roger82374** [OpenRadioss/OpenRadioss#2452] [Q&A]
Hi,

For example mydynajob.key will create an auto generated mydynajob_0001.rad file. This file contains the command /RUN//mydynajobkey/1/. The double forward slash "//" causes engine to fail. (Windows)

Many thanks

**Comment by sebastienVilleneuve** [comment]
@roger82374 

I let you join a small input deck , I will have a look ( I did not see this behavior before)

Thanks,
Sébastien

**Comment by elequiniou** [comment]
As there is no follow-up and it cannot be reproduced, we propose to move to discussion.
